content/it/docs/reference/glossary/cluster.md

```diff
@@ -4,14 +4,14 @@ id: cluster
 date: 2019-06-15
 full_link: 
 short_description: >
-   Un'insieme di macchine, chiamate nodi, che eseguono container e gestite da Kubernetes. Un cluster ha almeno un Worker Node e un Control Plane Node.
+   Un'insieme di macchine, chiamate nodi, che eseguono container gestiti da Kubernetes. Un cluster ha almeno un Worker Node e un Control Plane Node.
```

Comment:
Current English version has removed "e un Control Plane Node". I think we should match it.